cmd/fluxd/main.go

```diff
@@ -202,6 +202,7 @@ func main() {
 	fs.MarkDeprecated("k8s-namespace-whitelist", "changed to --k8s-allow-namespace, use that instead")
 	fs.MarkDeprecated("registry-poll-interval", "changed to --automation-interval, use that instead")
 	fs.MarkDeprecated("k8s-in-cluster", "no longer used")
+	fs.MarkDeprecated("git-verify-signatures", "changed to --git-verify-signatures-mode, use that instead")
```

Comment:
In addition to marking the flag as deprecated, setting the deprecated `--git-verify-signatures` flag should still result in `--git-verify-signatures-mode=all` below the surface; _but only_ if `--git-verify-signatures-mode` has not been set too.